etc/config/c++.amazon.properties

```diff
@@ -1594,6 +1594,13 @@ libs.highway.versions.trunk.path=/opt/compiler-explorer/libs/highway/trunk
 libs.highway.versions.0_12_2.version=0.12.2
 libs.highway.versions.0_12_2.path=/opt/compiler-explorer/libs/highway/0.12.2
 
+libs.hotels-template-library.name=Hotels Template Library
+libs.hotels-template-library.description=Hotels Template Library contains an open-source collection of C++ template libraries that are developed to make C++ development easier, safer and more efficient.
+libs.hotels-template-library.url=https://github.com/google/hotels-template-library
+libs.hotels-template-library.versions=trunk
+libs.hotels-template-library.versions.trunk.version=trunk
+libs.hotels-template-library.versions.trunk.path=/opt/compiler-explorer/libs/taojson/trunk/
```

Comment:
This seems to be pointing at `taojson`'s installation path. I guess this should be `/opt/compiler-explorer/libs/hotels-template-library/trunk/` ? Will merge and fix: thanks!